app/models/export/tasks/row.rb

```diff
@@ -12,7 +12,7 @@ def initialize(task)
       end
 
       def year_and_month
-        "#{task.period_year}#{format('%02d', task.period_month)}"
+        "#{task.period_year}-#{format('%02d', task.period_month)}"
```

Comment:
In the manually generated version this is reported as "YYYYMM", probably worth double checking this is actually what CCS want.

Reply:
Checked with @daibach, he's going to make sure the warehousing uses ISO8601.

Reply:
👍 agreed that we will use standard ISO8601 which includes hypens.